source/development_process/index.rst

```diff
@@ -0,0 +1,177 @@
+
+.. _osl-development-process:
+
+The OSL Development Workflow Guide
+======
+
+The OSL builds websites for a variety of clients including the Oregon Virtual
+School Districts, the Oregon Sea Grant Foundation, the OSU Biology department
+and Google. When the initial development is finished, the OSL will host and
+maintain the websites. This document will guide developers through the steps of
+creating a new website for a client. New developers are encouraged to read this
+document to understand how the developer team works, and experienced developers
+are encouraged to review this document to brush up their knowledge of parts of
+the process they may not be intimately familiar with. This workflow description
+will also be available to clients so they can understand how the OSL developer
+team works.
+
+How the Developer Team Interacts With Clients
+---------------------------------------------
+The needs of clients vary a surprising amount. The developer team is expected
+to modify the development and maintenance schedule to suit the specific needs
+of each customer. In the past, the OSL has adopted an agile methodology for
+working with the BSG and the Sea Grant foundation. This contrasts with the more
+conservative development and release cycle of the Protein Geometry Database,
+where every major change and new release must be certified by scientists.
+
+For the What's Fresh and Working Waterfronts website the developer team chose
+to follow the agile methodology. The agile method promotes quick responses to
+change and customer collaboration, which made it an ideal fit for working with
+both the Sea Grant Foundation and the BSG on an integrated website and mobile
+application. The agile development methodology is defined in `the agile
+manifesto <http://www.agilemanifesto.org/>`_, which is required reading for
```

Comment:
We've generally been leaving actual links as references after the paragraph they're used in. It makes for a much more readable block of text. i.e. this should be moved to::

```
.. _the agile manifesto: http://www.agilemanifesto.org
```